src/transaction/log_recovery.c

```diff
@@ -3998,6 +3999,7 @@ log_recovery_redo (THREAD_ENTRY * thread_p, const LOG_LSA * start_redolsa, const
 	    case LOG_DUMMY_HA_SERVER_STATE:
 	    case LOG_DUMMY_OVF_RECORD:
 	    case LOG_DUMMY_GENERIC:
+	    case LOG_SUPPLEMENTAL_INFO:
```

Comment:
I was wondering if applylogdb should also handle this log record type, so I checked the la_apply_log_file() and la_log_record_process() functions. But, I think we don't need to handle it. With this, do you have any opinion?